src/main/java/exam2/lotto/LottoPrize.java

```diff
@@ -0,0 +1,36 @@
+package exam2.lotto;
+
+public enum LottoPrize {
+    ZERO(0, 0),
+    ONE(1, 0),
+    TWO(2, 0),
+    THREE(3, 5_000),
+    FOUR(4, 50_000),
+    FIVE(5, 1_500_000),
+    SIX(6, 2_000_000_000);
+
+    private final int index;
+    private final int prize;
+
+    LottoPrize(int index, int prize) {
+        this.index = index;
+        this.prize = prize;
+    }
+
+    public int getIndex() {
+        return index;
+    }
+
+    public int getPrize() {
+        return prize;
+    }
+
+    public static int getPrize(int index) {
+        return LottoPrize.values()[index].getPrize();
+    }
+
+    public static LottoPrize getLottoPrize(int index) {
+        return LottoPrize.values()[index];
```

Comment:
```suggestion
           return Arrays.stream(LottoPrize.values())
                .filter((it) -> it.equalIndex(index))
                .findFirst()
                .orElse(LottoPrize.ZERO);
```
다음과 같이 작성하면 index의 위치가 변화 되더라도 대응이 가능할꺼 같아요!